tests/system/Database/Builder/FromTest.php

```diff
@@ -153,4 +153,18 @@ public function testFromSubqueryWithSQLSRV(): void
 
         $this->assertSame($expectedSQL, str_replace("\n", ' ', $builder->getCompiledSelect()));
     }
+
+    /**
+     * @see https://github.com/codeigniter4/CodeIgniter4/issues/8697
+     */
+    public function testConstructorWithMultipleSegmentTableWithSQLSRV(): void
+    {
+        $this->db = new MockConnection(['DBDriver' => 'SQLSRV', 'database' => 'test', 'schema' => 'dbo']);
+
+        $builder = new SQLSRVBuilder('database.dbo.table', $this->db);
```

Comment:
What if `dbo.table`?

Reply:
I will extend the code to add this scenario.

Reply:
Done.